packages/core/src/controllers/relayer.ts

```diff
@@ -268,6 +272,13 @@ export class Relayer extends IRelayer {
   }
 
   async transportOpen(relayUrl?: string) {
+    if (!this.subscriber.hasAnyTopics) {
+      this.logger.warn(
+        "Starting WS connection skipped because the client has no topics to work with.",
```

Comment:
should this be `warn` level or `debug`?

Reply:
got it. So we don't want folks to interact with this API? Can we test on AppKit just to make sure we don't get loads of warnings now unexpectedly?